README.md

```diff
@@ -46,7 +46,59 @@ filter-repo requires:
 
 `git-filter-repo` is a single-file python script, which was done to make
 installation for basic use on many systems trivial: just place that
-file into your $PATH.
+file into your `$PATH`. Example:
+
+### How to install `git filter-repo` on Linux Ubuntu:
```

Comment:
Thanks for the PR, but adding hundreds of sections ("How to install on Arch Linux", "How to install on Fedora Linux", "How to install on SuSE Linux", etc., then "How to install on Windows XP", "How to install on Windows 8", "How to install on Windows 11", then "How to install on Mac OS Ventura", "How to install on Mac OS Big Sur", etc., etc., etc.) makes absolutely no sense.  This isn't desirable.

Reply:
@newren , would you be willing to activate the community wiki for this repo? Then I'd just paste my info there.